lldb/source/API/CMakeLists.txt

```
@@ -147,7 +147,7 @@ endif()
 # BEGIN Swift Mods
 # Note that add_properties_for_swift_modules appends RPATHs so it's critical
 # that this is called after lldb_setup_rpaths.
-add_properties_for_swift_modules(liblldb)
+add_properties_for_swift_modules(liblldb "../../../../")
```

```suggestion
add_properties_for_swift_modules(liblldb "../../../../../../usr")
```

On Darwin, LLDB framework is installed at `<xctoolchain>/System/Library/PrivateFrameworks/LLDB.framework/Versions/A/LLDB`, so a few more hoops are required to get to the host dylibs

---

Ah, I was looking into the build directory structure. Thank you! I'll follow up since I jumped the gun on merging this.

---

https://github.com/apple/llvm-project/pull/5775